.devcontainer/docker-compose-vscode.yml

```diff
@@ -24,6 +24,8 @@ services:
       - SIM_V2X_PORT=5757
       - V2X_PORT=8686
       - INFRASTRUCTURE_ID=1
+      - SENSOR_JSON_FILE_PATH=/var/www/plugins/MAP/sensors.json
+      - KAFKA_BROKER_ADDRESS=127.0.0.1:9092
```

Comment:
We no longer need this configuration parameter. It is captured in the CARMA-Streets Plugin configuration.

Reply:
Addressed